packages/e2e-tests/specs/blocks/group.test.js

```diff
@@ -25,7 +24,21 @@ describe( 'Group', () => {
 		await clickBlockAppender();
 		await page.keyboard.type( '/group' );
 		await page.keyboard.press( 'Enter' );
-		await page.keyboard.type( 'Group block' );
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	it( 'can have other blocks appended to it using the button appender', async () => {
+		await searchForBlock( 'Group' );
+		await page.click( '.editor-block-list-item-group' );
+
+		await page.waitForSelector( '.block-editor-button-block-appender' );
```

Comment:
It's a bit concerning to me that we have to wait. Why isn't it immediately available?

Reply:
I think I'm feeling a little bit burned by failing tests and go with the safest option. 😄 

I've removed those calls and tests still pass.